src/editors/data/redux/video/reducer.js

```diff
@@ -41,23 +41,6 @@ const video = createSlice({
     load: (state, { payload }) => ({
       ...payload,
     }),
-    addTranscript: (state, { payload }) => ({
```

Comment:
confused by these changes

Reply:
did we somehow mess up and get bad code in the last PR?

Reply:
They were left over variables that we forgot to delete